mac/build.sh

```diff
@@ -12,80 +12,44 @@ KEYMAN_MAC_BASE_PATH="$KEYMAN_ROOT/mac"
 . "$KEYMAN_ROOT/resources/build/build-help.inc.sh"
 . "$KEYMAN_ROOT/mac/mac-utils.inc.sh"
 
-# This script runs from its own folder
-cd "$(dirname "$THIS_SCRIPT")"
-
+builder_describe "Builds Keyman for macOS." \
+  "@/core:mac" \
+  "clean" \
+  "configure" \
+  "build" \
+  "publish   Publishes debug info to Sentry and builds DMG, .download_info for distribution" \
+  "test" \
+  "install   Installs result of Keyman4MacIM locally." \
+  ":engine   KeymanEngine4Mac" \
+  ":im   Keyman4MacIM" \
+  ":testapp   Keyman4Mac (test harness)" \
+  "--quick,-q  Bypasses notarization for $(builder_term install)" 
```

Comment:
Exists to provide the `local` vs `quicklocal` distinction.  That said, I suspect we should drop this in favor of simplicity, basing the distinction based on presence of `--debug`.  (See a different comment for more details + request for guidance in the matter.)